packages/server-web/package.json

```diff
@@ -50,6 +50,7 @@
     "passport-local": "^1.0.0",
     "vue": "^2.5.16",
     "vue-resource": "1.5.1",
-    "vue-router": "^3.0.1"
+    "vue-router": "^3.0.1",
+    "vuex": "^3.0.1"
```

Comment:
Umm, when we use `vue-router` and `vuex` in version 3+, do we need to use vue 3+ as well or are they not related in version numbers?

Reply:
https://router.vuejs.org/ says, vue-router 3.0+ requires vue 2.5+
Couldn't find any other information on it, except an issue on another of vue's projects where someone asks the same question but doesn't get a real answer https://github.com/vuejs/vuex-router-sync/issues/57